src/categories/views/CategoryDetails.tsx

```diff
@@ -210,166 +211,179 @@ export const CategoryDetails: React.FC<CategoryDetailsProps> = ({
   return (
     <>
       <WindowTitle title={maybe(() => data.category.name)} />
-      <CategoryUpdatePage
-        channelsCount={availableChannels.length}
-        channelChoices={channelChoices}
-        changeTab={changeTab}
-        currentTab={params.activeTab}
-        category={maybe(() => data.category)}
-        disabled={loading}
-        errors={updateResult.data?.categoryUpdate.errors || []}
-        onAddCategory={() => navigate(categoryAddUrl(id))}
-        onAddProduct={() => navigate(productAddUrl())}
-        onBack={() =>
-          navigate(
-            maybe(() => categoryUrl(data.category.parent.id), categoryListUrl())
-          )
-        }
-        onCategoryClick={id => () => navigate(categoryUrl(id))}
-        onDelete={() => openModal("delete")}
-        onImageDelete={() =>
-          updateCategory({
-            variables: {
-              id,
-              input: {
-                backgroundImage: null
-              }
+      {typeof channel !== "undefined" ? (
```

Comment:
Nope, because `undefined` means that channels are not loaded yet and `null` means that there are no channels.

`!channels === (channels === null || channels === 'undefined')`, but `null !== 'undefined'`